.github/workflows/behat-stability-1-basic-installation.yml

```diff
@@ -0,0 +1,50 @@
+name: 'Behat Basic Installation'
+
+# Controls when the workflow will run
+on:
+  pull_request:
+    types: [ ready_for_review, reopened, synchronize ]
```

Comment:
Let's add documentation pointing to triggers and webhook events.

https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request
https://docs.github.com/en/developers/webhooks-and-events/webhooks/webhook-events-and-payloads#webhook-payload-object-35

Reply:
Also, by the definitions given, synchronize is a default action. So, it may land at the same situation which we are trying to add checks.